Automattic/wp-calypso
# Removing the redirects for premium theme pages

#### Changes proposed in this Pull Request

More info: pcbrnV-4gf-P2

This change removes the redirects from /themes/premium to /themes, plus any theme route which contains a type parameter (ie. free or premium).

#### Testing instructions

Visit these URLs and ensure that there is no redirect in place:
- /themes/premium 
- /es/themes/premium 
- /themes/portfolio/free
- /themes/portfolio/premium/filter/author-bio

#### Related to 56430
